Automattic/wp-calypso
# Jetpack Blocks: Add the jetpack contact form blocks to the preset

#### Changes proposed in this Pull Request

* Add the contact form presets previously missing. 

#### Testing instructions
* Build this PR in Jetpack and .com does things work as expected? 
* Are you able to load create the contact form  as expected?

We missed this in the previous release and allowed all the blocks to be loaded as they were.
